examples/03_connectivity/plot_seed_based_connectome.py

```diff
@@ -1,57 +1,89 @@
 """
-Extract signals and plot a connectome for seed-based atlases
-============================================================
+Extract signals and plot a connectome for atlas-based spheres
```

Comment:
Actually, I think that the word atlas should be in the title. How about "Extract signals on spheres from an atlas and plot a connectome"